firebase/initFirebase.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-unused-vars */
+import { initializeApp } from 'firebase/app';
+import { getFirestore } from 'firebase/firestore';
+// import { getAnalytics } from 'firebase/analytics';
+
+// TODO: Add SDKs for Firebase products that you want to use
+
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+
+const firebaseConfig = {
+  apiKey: 'AIzaSyCDvvTB9u26lnFzwJjFNZJ6OGXUX_XxdFE',
+
+  authDomain: 'wiki-d6b95.firebaseapp.com',
+
+  projectId: 'wiki-d6b95',
+
+  storageBucket: 'wiki-d6b95.appspot.com',
+
+  messagingSenderId: '589888164658',
+
+  appId: '1:589888164658:web:cfa3094ae2d6337823fd2b',
+
+  measurementId: 'G-DLDL7X8BP8',
```

Comment:
These will definitely need to be out. You should not add your production API key to code :)) 

https://nextjs.org/docs/basic-features/environment-variables